src/ChatGptNet/Models/ChatGptRequest.cs

```diff
@@ -74,4 +74,12 @@ public class ChatGptRequest
     /// </remarks>
     [JsonPropertyName("frequency_penalty")]
     public double? FrequencyPenalty { get; set; }
+
+    /// <summary>
+    /// Gets or sets unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse.
```

Comment:
For the sake of consistency, it's better to use the same summary that you have written in the `ChatGptOptions` class.

Reply:
ofc, my bad